main.py

```
@@ -383,33 +439,23 @@ def calibrate_from_frames(frames, board, dictionary, detector, charuco_detector,
+    for i in range(len(all_charuco_corners)):
+        try:
+            reprojected, _ = cv2.projectPoints(
+                board_obj_points[all_charuco_ids[i]],
+                rvecs[i], tvecs[i],
+                camera_matrix, dist_coeffs
+            )
+            error = cv2.norm(all_charuco_corners[i], reprojected, cv2.NORM_L2) / len(reprojected)
+            mean_error += error
+            errors_per_frame.append(float(error))
+        except Exception:
+            continue
```

Ruff (0.14.14)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)